--- ./src/core/engine.cpp	2017-07-02 09:07:35.032616272 +0100
+++ ../BUILD/knewstuff-5.36.0/src/core/engine.cpp	2017-08-28 00:35:09.280595930 +0100
@@ -118,12 +118,6 @@
 
     emit signalBusy(i18n("Initializing"));
 
-    if(!QFileInfo::exists(configfile)) {
-        emit signalError(i18n("Configuration file not found: \"%1\"", configfile));
-        qCritical() << "No knsrc file named '" << configfile << "' was found.";
-        return false;
-    }
-
     KConfig conf(configfile);
     if (conf.accessMode() == KConfig::NoAccess) {
         emit signalError(i18n("Configuration file exists, but cannot be opened: \"%1\"", configfile));
